tyk-stack/README.md

```diff
@@ -258,158 +440,173 @@ To set up other backends for pump, refer to this [document](https://github.com/T
+      ##     mountPath: /etc/ssl/certs/ca-certs.crt
+      ##     readOnly: true
+      extraVolumeMounts: []
+      mounts: []
+  
+      # Dashboard will only bootstrap if the master bootstrap option is set to true.
+      bootstrap: true
+  
+      # The hostname to bind the Dashboard to.
+      hostName: tyk-dashboard.local
+      # If set to true the Dashboard will use SSL connection.
+      # You will also need to set the:
+      # - TYK_DB_SERVEROPTIONS_CERTIFICATE_CERTFILE
+      # - TYK_DB_SERVEROPTIONS_CERTIFICATE_KEYFILE
+      # variables in extraEnvs object array to define your SSL cert and key files.
+      tls: false
+  
+      # Dashboard admin information.
+      adminUser:
+        firstName: admin
+        lastName: user
+        email: default@example.com
+        # Set a password or a random one will be assigned.
+        password: "123456"
+      # Dashboard Organisation information.
 ```
+### Tyk Developer Enterprise Portal Configurations
 
-> [!NOTE]
-> Once `global.secrets.useSecretName` is declared, it takes precedence over `global.secrets.APISecret` and `global.secrets.AdminSecret`.
-
-#### Dashboard License
-
-In order to refer Tyk Dashboard license through Kubernetes secret, please use `global.secrets.useSecretName`, where 
-the secret should contain a key called `DashLicense`.
-
-#### Developer Portal License
-
-In order to refer Tyk Developer Portal license through Kubernetes secret, please use `global.secrets.useSecretName`, 
-where the secret should contain a key called `DevPortalLicense`.
-
-#### Developer Portal Admin Password
-
-In order to refer Tyk Developer Portal's admin password through Kubernetes secret, 
-please use `global.adminUser.useSecretName`, where the secret should contain a key called `adminUserPassword`.
-
-#### Developer Portal Storage Connection String
-
-In order to refer Tyk Developer Portal connection string to the selected database through Kubernetes secret,
-please use `global.secrets.useSecretName`, where the secret should contain a key called 
-`DevPortalStorageConnectionString`.
-
-> [!WARNING]
-> If `global.secrets.useSecretName` is in use, please add all keys mentioned above to the secret. 
-
-#### Remote Control Plane Configuration
+To enable Tyk Developer Enterprise Portal, set `global.components.devPortal` to true, and configure below inside `tyk-dev-portal` section.
 
-All configurations regarding remote control plane (`orgId`, `userApiKey`, and `groupID`) can be set via
-Kubernetes secret.
+#### Tyk Developer Enterprise Portal License (Required)
 
-Instead of explicitly setting them in the values file, just create a Kubernetes secret including `orgId`, `userApiKey`
-and `groupID` keys and refer to it in `global.remoteControlPlane.useSecretName`.
+Tyk Developer Enterprise Portal License is required. It can be set up in `tyk-dev-portal.license` or through secret `global.secrets.useSecretName`. The secret should contain a key called DeveloperPortalLicense.
```

Comment:
```suggestion
Tyk Developer Enterprise Portal License is required. It can be set up in `tyk-dev-portal.license` or through secret `global.secrets.useSecretName`. The secret should contain a key called `DevPortalLicense`.
```